rust/examples/otel-stdout.rs

```diff
@@ -39,11 +39,12 @@ pub async fn main() -> anyhow::Result<()> {
         .get_func(&mut store, function_name)
         .expect("function exists");
 
-    trace_ctx.set_trace_id("any-old-trace-id".to_string()).await;
+    trace_ctx.set_trace_id(0u64).await;
```

Comment:
I switched these to u64. We should either switch all ids to u64 or come up with a more generic way to handle it.